### DISCUSSION TITLE
Bots keep on running around in circles.

### DISCUSSION THREAD

**Discussion by darcochoco** [Joe91/fun-bots#201] [Q&A] [answered]
In certain maps, the friendly bots like to circle around non-stop and shooting each other when there's collision. They dont leave the spawn area for example at Epicenter, Azadi Palace, Armored Shield maps etc. 

How to fix this AI behavior?

**Comment by Joe91** [comment]
Thanks for that feedback.
Can you just list all maps and gamemodes this happens for you? Then we can fix these paths...

**Comment by darcochoco** [comment]
List of maps: I'm playing 12 v 12 bots in TDM in various maps
1) Azadi Palace TDM (spawn base outside the palace in an open area)
2) Epicenter TDM (spawn base where you have to blow up the flammable canister to clear the doors)
3) Alborz Mountains TDM (merry go round usually on top of the tower)
4) Sharqi Peninsula TDM (same as Alborz mountain, but on top of a tall building)
4) Armored Shield TDM (spawn base inside one of the buildings)
5) Seine Crossing TDM (running around in circles around the main building with multiple floors and enemy BOTs rarely leave their spawn base at the opposite building across the main road)
6) Markaz Monolith TDM (inside a building with wall glasses near the junk cars just outside the central building)
7) Caspian Border TDM (friendly bot just running around in the safe zone with very little combat happening, I have to snipe enemy AI if not the game would never finish).
8) Grand Bazaar TDM (there's like a tunnel entering the Market area where they just stood there and move back and forth)
9) Operation Metro TDM (merry go round inside the bar near the umbrella tents, facing the bus)
10) Operation 925 TDM CQ (in one of the twin washrooms near the main staircase and big lamps hanging from the ceiling)

Another issue I found:

I'm being the main target for bots and most of the time my team doesn't shoot the nearby enemy and vice versa. They just let them pass by for free. Combat initiation  needs some fixing

Also i wish the friendly bots would use 'Spot Enemy' ability  and shoot down the enemies I spotted.

**Comment by Joe91** [comment]
In the new dev-versions the bot-bot attack should be much better. But I have to focus on the player for the attack to keep the game responsive.
If you want you can also try to fix the TDM paths on your own. Not that hard. Just delete the problematic paths and create some new ones... Those were the fist paths that existed so they really might be not that up to date...

**Reply by darcochoco** [reply]
Hey Joe,
I see. However, I don't know exactly (step-by-step)  how to fix the TDM paths so I was hoping all these issues will be fixed in the upcoming release. I can wait and just play the other maps until the new version comes out.

**Reply by Joe91** [reply]
All fine. I already started with them ;-).
When you find more issues just let me know...

**Comment by Joe91** [comment, marked as answer]
Think I fixed all of them for now. Will create a new dev-version soon...

**Reply by darcochoco** [reply]
Glad to hear that. Thanks a lot.
